### ISSUE TITLE
GI Bill CT: SEO needs improvement

### ISSUE THREAD

**Issue by saintsoup52** [department-of-veterans-affairs/vets-website#5889]
Earlier this week the RAND Corporation came out with a report about the Post-9/11 GI Bill.

https://www.rand.org/pubs/research_reports/RR1766.html  

The most interest part of the report to me was the word cloud showing what servicemembers and veterans are google searching when they are researching the GI Bill. “How much is the gi bill” was the most searched GI Bill phrase last year, which has been our working theory with the Comparison Tool all along.

![image](https://user-images.githubusercontent.com/4397508/27638264-a654bea8-5be0-11e7-8fd1-c18cc42e0836.png)

Sadly, if you google "how much is the gi bill" right now…the first three links are to non-VA sites and google highlights information from a non-VA site about the Montgomery GI Bill ironically with a picture of the Post-9/11 GI Bill. 

 
![image](https://user-images.githubusercontent.com/4397508/27638297-bf732e2e-5be0-11e7-8e9a-5c0b8fe2f400.png)

And the GI Bill comparison tool is way down the list and the subtext is the legal disclaimer which does nothing to explain why a user should click on this link versus the 10 above it.

 
![image](https://user-images.githubusercontent.com/4397508/27638301-c538bf7c-5be0-11e7-936c-b4f5e5c9df16.png)

What can we do to improve the GI Bill Comparison Tool's SEO... especially for this search phrase?

**Comment by saintsoup52** [comment]
Sorry forgot to tag folks @leannammiller @melwoodard

**Comment by ayaleloehr** [comment]
Great point @saintsoup52 . Also tagging @robertfairhead and @suzchap as I know this topic is on your radar as well. This is a high traffic page, so could be a good candidate to focus on first for SEO.

**Comment by robertfairhead** [comment]
We can take some basic steps to improve SEO on the pages around metadata and descriptions. I'm still catching up from being out for my move, but I can do that next week. I'm assigning myself.

@leannammiller let me know if there's someone who i should use to review the PR when it's done.

**Comment by robertfairhead** [comment]
I took at a look at this today. I opened a PR that'll link to this that adds description tags and fills in one for the GIBCT landing page. That description is our "suggestion" to Google, Bing, etc on what to use as the short description. Ultimately they will use whatever they want, but that tag is the best way to suggest clear language. I pulled the subhead text from the landing page to use but if there's a preferred alternative, feel free to drop that in the PR.

I checked our Google search data with respect to that search phrase over the past three months. It was the 50th query for us in terms of impressions (e.g., when someone saw our link) for queries that included the term `gi bill`. Our SEO position is on the first page of results for that so nearly all searches should result in an impression. I can't speak to Rand's methodology (not having seen their paper or data first hand), but I would be leery of over optimizing our site against that phrase based on our direct Google search data.

[Analytics All Vets.Gov Site Data Queries 20170401-20170629.pdf](https://github.com/department-of-veterans-affairs/vets-website/files/1120887/Analytics.All.Vets.Gov.Site.Data.Queries.20170401-20170629.pdf)

To improve our search ranking, the best course would be to increase the number of inbound links from other high-ranking sources. In our case, that would likely be other relevant va.gov sites. The more of them that link into the GI Bill Comparison Tool using relevant anchor text, the higher our ranking will get.
